values.go

```diff
@@ -786,3 +929,19 @@ func (v Event) ToGoValue() interface{} {
 
 	return ret
 }
+
+func (v Event) ToRuntimeValue() runtime.Value {
+	return compositeToRuntimeValue(v.EventType.Fields, v.Fields)
+}
+
+func compositeToRuntimeValue(fieldTypes []Field, fieldValues []Value) runtime.Value {
+	fields := make(map[string]interpreter.Value, len(fieldTypes))
+
+	for i := 0; i < len(fieldTypes) && i < len(fieldValues); i++ {
+		fieldType := fieldTypes[i]
+		fieldValue := fieldValues[i]
+		fields[fieldType.Identifier] = fieldValue.ToRuntimeValue().Value
+	}
+
+	return runtime.Value{Value: &interpreter.CompositeValue{Fields: fields}}
```

Comment:
Just the fields are unfortunately not enough, `Location`, `TypeID`, and `Kind` are needed as well: This information is used e.g. to link in the composite value's functions